src/components/dds/MessageBar/MessageBar.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import type { ReactNode } from 'react';
+import getRootClassNames from '../common/getRootClassNames';
+import { componentClasses } from 'dds-core/components/message-bar/ts/messagebar';
+import { Icon } from '../Icon';
+import 'dds-core/scss/dds-main.scss';
+
+export type MessageBarAlignment = 'center';
+export type MessageBarVariant = 'success' | 'warning' | 'error';
+export type MessageBarLayout = 'global';
+
+export interface MessageBarProps extends React.HTMLAttributes<HTMLDivElement> {
+    /**
+     * Action link to be displayed in the message bar
+     * */
+    actionLink?: ReactNode;
+    /**
+     * Changes the alignment of the content of the message bar
+     * */
+    alignment?: MessageBarAlignment;
+    /**
+     * Allows different types of MessageBar
+     * */
+    variant?: MessageBarVariant;
+    /**
+     * Toggles the message bar's ability to be dismissible
+     * */
+    dismissible: boolean;
+    /**
+     * Icon at the beginning of the message bar
+     * */
+    icon?: boolean;
+    /**
+     * Layouts allow for different usages of message bars
+     * */
+    layout?: MessageBarLayout;
+    /**
+     * Text label for the input
+     * */
+    title?: string;
+    /**
+     * Whether the MessageBar is visible
+     * */
+    open: boolean;
+    /**
+     * Action executed on close button click
+     * */
+    onClose?: () => void;
+    /**
+     * MessageBar content
+     * */
+    children: ReactNode;
+}
+
+/**
+ * React implementation of DDS MessageBar component
+ */
+export function MessageBar({
+    actionLink,
+    alignment,
+    variant,
+    dismissible = true,
+    layout,
+    icon,
+    title = 'Notice: ',
+    open,
+    onClose,
+    children
+}: MessageBarProps) {
+    const handleClose = () => {
+        if (onClose) {
+            onClose();
+        }
+    };
```

Comment:
Optional:

Probably shorter version will also work here 🙂 
```suggestion
    const handleClose = () => onClose?.();
```

Reply:
done: https://github.com/cloudify-cosmo/cloudify-ui-components/pull/240/commits/72cfc74ac6281101b3c253445da2b1797c56b64e